.github/workflows/maintainer_merge_comment.yml

```diff
@@ -27,8 +27,9 @@ jobs:
         id: determine_topic
         run: |
           ./scripts/get_tlabel.sh "${PR}" >> "$GITHUB_OUTPUT"
+        with:
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
This needs to be back where it was below in the `env` section to work. (Same for the other instances.)